hooks/environment

```diff
@@ -12,5 +12,6 @@ PLUGIN_ROOT=$( dirname "$HOOKS_ROOT" )
 PLUGIN_BIN="$PLUGIN_ROOT/bin"
 
 export PATH="$PATH:$PLUGIN_BIN"
+export PLUGIN_ROOT=$PLUGIN_ROOT
 
 echo "~~~ :file_cabinet: Loaded Caching Plugin"
```

Comment:
I think this is a leftover from when this environment hook was used to load our git cache plugin (back when it was named bash-cache instead of a8c-ci-toolkit?) or something like that … or maybe it was just copied from the Buildkite plugin's own hook, hence why the env vars mention "PLUGIN" in their name?

Anyway, from the looks of the content of this hook nowadays, this seem to be used for a different usage (or is it even really needed at all, vs deriving those paths from the repo root directly or some existing env var already exposed by Buildkite, given that if I understand the logic correctly `PLUGIN_ROOT` will end up resolving to the root of the repo?), so at best this `echo` is misleading and should be updated to reflect what is done here, or the hook file is obsolete and should be removed completely…

Reply:
I see, hm WDYT about creating an issue to review if `hooks/environment` is needed at all? I'd prefer to not handle this it in this PR 😅